gulpfile.js

```diff
@@ -154,6 +155,20 @@ function copyFactory(resourceName, sourceFolder, targetFolder) {
 
 }
 
+function copyFiletypeFactory(resourceName, sourceFolder, sourceExtension, targetFolder) {
```

Comment:
`sourceFileExtension` would be a little bit more readable

Reply:
We already have `copyFileFactory` that copies all files, and `copyFiletypeFactory` seems a logical extension.  Could be `copyFilesWithExtensionFactory`? But `sourceFileExtension` doesn't mean a lot to me.

Reply:
that is for the argument name rather than the function name (if that wasn't clear)

Reply:
Yeah, I updated the argument name, made sense